packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -843,12 +843,18 @@ class VmServiceWrapper implements VmService {
 
   Future<HttpProfileRequest> getHttpProfileRequest(
     String isolateId,
-    int id,
+    String id,
   ) async {
     assert(await isHttpProfilingAvailable(isolateId));
     return trackFuture(
       'getHttpProfileRequest',
-      _vmService.getHttpProfileRequest(isolateId, id),
+      _vmService.callMethod(
```

Comment:
Leave a TODO here to switch to the actual implementation once we roll forward to v11.